db_setup.sql

```diff
@@ -0,0 +1,57 @@
+
+CREATE SEQUENCE transactions_seq
+  start 1
+  increment 1;
+
+CREATE SEQUENCE customers_seq
+  start 1
+  increment 1;
+ 
+CREATE SEQUENCE products_seq
+  start 1
+  increment 1;
+ 
+CREATE TABLE customers (
+	"id" int NOT null default nextval('customers_seq'),
+	"name" VARCHAR(255) NOT NULL,
+	CONSTRAINT "customers_pk" PRIMARY KEY ("id")
+);
+
+CREATE TABLE products (
+	"id" int NOT null default nextval('products_seq'),
+	"name" VARCHAR(255) NOT NULL,
+	CONSTRAINT "products_pk" PRIMARY KEY ("id")
+);
+
+CREATE TABLE transactions (
+	"id" int NOT null default nextval('transactions_seq'),
+	"customer_id" integer NOT NULL,
+	"product_id" integer NOT NULL,
+	"price" DECIMAL NOT NULL,
+	"quantity" integer not null,
+	"created_at" TIMESTAMP NOT null DEFAULT now(),
+	CONSTRAINT "transactions_pk" PRIMARY KEY ("id")
+); 
+
+ALTER TABLE transactions ADD CONSTRAINT "transactions_fk0" FOREIGN KEY ("customer_id") REFERENCES "customers"("id");
+ALTER TABLE transactions ADD CONSTRAINT "transactions_fk1" FOREIGN KEY ("product_id") REFERENCES "products"("id");
+
+insert into customers (name) values('Abdulrahman');
```

Comment:
separate seed data in a different file.

Reply:
i got the point, but ill keep them in one file to make project initialization easier for others.